src/app.php

```diff
@@ -50,7 +50,7 @@
 });
 
 $app['auth.controller'] = function () use ($app) {
-    return new \Controller\AuthController($app['twig']);
+    return new \Controller\AuthController($app['twig'], $app['form'], new \Controller\LoginFormController($app['form.factory']));
```

Comment:
constructor przyjmuje tylko 2 argumenty a ty przekazujesz 3

nie masz zdefiniowane $app['form'] wiec silex nie wie co to jest 'form'

musisz zdefiniowac $app['form'] = $app['form.factory']->createBuilder(FormType::class, $data) ....